nibabel/tests/test_nifti1.py

```diff
@@ -578,12 +578,12 @@ def test_slice_times(self):
         with pytest.raises(HeaderDataError):
             # all None
             hdr.set_slice_times((None,) * len(times))
-        n_mid_times = times[:]
+        n_mid_times = times.copy()
```

Comment:
Debatable, as `[:]` has been a Python idiom for such a long time, but I do find the English word `copy` more readable, especially for non-Python programmers.